pkg/kv/kvnemesis/generator.go

```diff
@@ -424,6 +424,7 @@ type FaultConfig struct {
 	// RestartNode is an operation that restarts a randomly chosen node.
 	RestartNode int
 	// Disk stalls and other faults belong here.
+	CrashServer int
```

Comment:
nit: `CrashNode`, to match the naming above? Also, comment it similarly to `StopNode`.